public/css/styles.css

```diff
@@ -0,0 +1,58 @@
+body{
+  background-color: white;
+  font-weight: 600;
+  
+}
+.links{
+  text-decoration: none;
+}
+.season{
+  text-align: center;
+}
+.headerColor{
+  background-color: #6F91EB;
+  color: white;
+}
+
+.episode{
+  border: 1px solid black;
+  margin: 10px;
+  padding: 8px;
+}
+
+.number{
+  width: 5%;
+  border: 1px solid black;
+  margin: 1px;
+  padding: 1px;
+  text-align: center;
+}
+
+.synopsis{
+  width: 100%;
+  border: 1px solid black;
```

Comment:
You could have pulled some of this CSS out into the table like the border rather than having it be repetitive.